<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView::item:selected {
    background-color: rgb(100, 181, 246);
}
QTableView::item:selected:!focus {
    background-color: rgb(187, 222, 251);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="SearchText">
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="SearchBy">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="placeholderText">
         <string>-- 选择查询方式 --</string>
        </property>
        <item>
         <property name="text">
          <string>姓名</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>学号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>班级</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btSearch">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btAdd">
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btDelete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btEdit">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="OrderBy">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="placeholderText">
         <string>-- 选择排序方式 --</string>
        </property>
        <item>
         <property name="text">
          <string>学号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>姓名</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>性别</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>班级</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>数学成绩</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C语言成绩</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Java成绩</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>总分</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="AscOrDesc">
        <property name="text">
         <string>升序</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btOrder">
        <property name="text">
         <string>排序</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
